_topics/en/op_checktemplateverify.md

```diff
@@ -131,30 +131,230 @@ see_also:
+  outputs.  When the initial transaction is confirmed to a suitable
+  depth, this can assure the parties with outputs in the follow-up
+  transaction that they will eventually be paid (e.g. when feerates are
+  lower).  This two-step process can probably be used anywhere [payment
+  batching][topic payment batching] is an option, although it requires
+  extra communication to receivers for them to learn that they have been
+  paid prior the follow-up transaction being confirmed.
+
+- **Channel factories:** each change to a [channel factory's][topic
+  channel factories] top-level state requires each party to receive a
+  unilateral exit transaction that refunds them their current balance in
+  the factory in case other parties later become unresponsive.  Without
+  CTV or something similar, unilateral exit transactions must be signed
+  by all participants in a channel factory before each member signs the
+  new state transaction, requiring two full rounds of communication.
+  With CTV, a commitment to the refund transaction can be included
+  directly in the next state transaction.  This allows any party to
+  propose a state transition that only requires one round of signing.
+  Channel factories may also benefit from other uses of CTV, such as
+  allowing channels to be opened in congestion control transactions.
+  Channel factories have a similar structure to joinpools, so any
+  features of CTV joinpools described below may also apply to CTV
+  channel factories.
+
+- **Joinpools:** similar to a channel factory, multiple users can [pool
+  their funds together][topic joinpools] by creating a UTXO that allows
+  any of them to unilaterally withdraw their funds at any time or allows
+  all of them in agreement to spend a portion of the funds.  This is
+  possible today by having each participant sign the unilateral exit
+  transaction for the next state before they sign the next state, but
+  this requires two rounds of signing.  When the number of parties
+  needed to sign is large, each round can take a long time.  CTV reduces
+  the amount of signing to one round.
+
+  Additionally, when one party wants to unilaterally exit a joinpool or
+  channel factory, some other parties may want to remain in the joinpool
+  (or form subsets of the original joinpool).  This can minimize the
+  amount of onchain fees that need to be paid by each party.  Remaining
+  in an existing joinpool requires each party agree to the outcomes for
+  each other party exiting.  Using presigned exits, the most efficient
+  onchain version of this requires generating and transmitting O(n!)
+  offchain signatures for _n_ parties, making it impractical for more
+  than a few users to participate in any individual joinpool with
+  per-party exit paths.  Using CTV, each signature operation can be
+  replaced with a CTV operation, which leaves the computational
+  complexity but doesn't require transmitting any additional data---each
```

Comment:
I'm not 100% sure this description is accurate.  I need to do some additional research or rewrite it to be less specific before we publish.

Reply:
probably makes sense to discuss ARK and ARK v2 here, which AFAIU both work under CTV as a standalone upgrade?

Reply:
Made text vague about the actual complexity (to address my point) and added back link from our Ark topic page to this CTV to partially address @JeremyRubin's point.  I forgotten most about what I learned about Ark last year and haven't learned anything about Ark v2 yet, and I don't want to block updating this page on learning about them to maybe add a sentence or two.